src/screens/ContactScreen/index.js

```diff
@@ -262,9 +326,8 @@ const styles = StyleSheet.create({
   containerStyle: {
     marginHorizontal: 16
   },
-  subtitleStyle: (selectedUsers) => ({
-    color: selectedUsers.length > 0 ? COLORS.anon_primary : COLORS.gray4,
-    marginEnd: 8
+  subtitleStyle: (isAnon) => ({
+    color: isAnon ? COLORS.anon_primary : COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `subtitleStyle` now depends on the `isAnon` state variable instead of `selectedUsers`. This is a good practice as it allows the style to be updated based on the context it is used in.
```diff
+ subtitleStyle: (isAnon) => ({
+   color: isAnon ? COLORS.anon_primary : COLORS.signed_primary
+ })
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->